### PR TITLE
Hw 26 2

### PR DESCRIPTION
**Pull request by andromash**

### CODE REVIEW

**Review by dariiabutorina — APPROVED** [review]
Nice try :D

**Review by dariiabutorina — APPROVED** [review]
Nice try :D

File: src/main/java/com/internet/shop/controllers/order/GetOrderByIdController.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+import java.io.IOException;
+import java.math.BigDecimal;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/user/order")
+public class GetOrderByIdController extends HttpServlet {
```
Comment (RomanDubovskyi) [inline src/main/java/com/internet/shop/controllers/order/GetOrderByIdController.java]:
```suggestion
public class GetOrderDetailsController extends HttpServlet {
```

File: src/main/java/com/internet/shop/controllers/user/GetAllUsersOrderController.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+import java.io.IOException;
+import java.util.List;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/user/orders")
+public class GetAllUsersOrderController extends HttpServlet {
```
Comment (RomanDubovskyi) [inline src/main/java/com/internet/shop/controllers/user/GetAllUsersOrderController.java]:
```suggestion
public class GetUsersOrdersController extends HttpServlet {
```

**Review by RomanDubovskyi — COMMENTED** [review]
all good, let's just edit naming of controllers a bit

File: src/main/java/com/internet/shop/controllers/shoppingcart/DeleteItemFromTheCartController.java [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+import com.internet.shop.service.ShoppingCartService;
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/shopping-cart/product/delete")
+public class DeleteItemFromTheCartController extends HttpServlet {
```
Comment (dmytro-hryhoruk) [inline src/main/java/com/internet/shop/controllers/shoppingcart/DeleteItemFromTheCartController.java]:
```suggestion
public class DeleteProductFromTheCartController extends HttpServlet {
```

**Review by dmytro-hryhoruk — APPROVED** [review]

**Pull request merged**